presto-main/src/main/java/com/facebook/presto/operator/AggregationOperator.java

```diff
@@ -14,7 +14,6 @@
 package com.facebook.presto.operator;
 
 import com.facebook.presto.memory.context.LocalMemoryContext;
-import com.facebook.presto.operator.aggregation.Accumulator;
```

Comment:
[Optiopnal] It would be great to add some benchmarks as we have for `BenchmarkHashBuildAndJoinOperators`

Reply:
@sopel39 Karol, I created `BenchmarkStreamingAggregationOperator` using `BenchmarkHashBuildAndJoinOperators` as an example. The benchmark processes 1.4M rows split into pages 10K rows each. `rowsPerGroup` parameter determines the number of rows with the same grouping key. Here are the results for 1, 10 and 1,000 `rowsPerGroup`:

```
# Run complete. Total time: 00:04:25

Benchmark                                        (rowsPerGroup)  Mode  Cnt    Score   Error  Units
BenchmarkStreamingAggregationOperator.benchmark               1  avgt   30  375.009 ± 6.488  ms/op
BenchmarkStreamingAggregationOperator.benchmark              10  avgt   30   66.805 ± 1.929  ms/op
BenchmarkStreamingAggregationOperator.benchmark            1000  avgt   30   28.444 ± 0.346  ms/op
```

Processing larger groups is significantly faster than processing small groups. I'm going to explore your idea of reusable accumulators next.